README.md

```diff
@@ -137,6 +137,7 @@ $ kubectl exec -it samplepod -- ip a
 - type (string, required): &quot;multus&quot;
 - kubeconfig (string, optional): kubeconfig file for the out of cluster communication with kube-apiserver. See the example [kubeconfig](https://github.com/intel/multus-cni/blob/master/doc/node-kubeconfig.yaml)
 - delegates (([]map,required): number of delegate details in the Multus
+- capabilities ({}list, optional): [capabilities](https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md#dynamic-plugin-specific-fields-capabilities--runtime-configuration) supported by at least one of the delegates. (NOTE: Mutlus only supports portMappings capability for now). See the [example](https://github.com/intel/multus-cni/blob/master/examples/multus-ptp-portmap.conf)
```

Comment:
Nit: change `Mutlus` to `Multus` (thanks for updating docs!)

Reply:
done